src/resources/postcss/tickets-admin/settings/_order_modifiers.pcss

```diff
@@ -26,3 +26,16 @@
 		width: 30%;
 	}
 }
+
+.tickets_page_tec-tickets-order-modifiers {
+	.event-tickets__order-modifiers-admin-form {
+		.tec-tickets__modifier-page-title {
+			font-weight: var(--tec-font-weight-regular);
+
+			& > a {
```

Comment:
This will affect all links in `.tec-tickets__modifier-page-title {` - intentional? Or do we want to lock it down to just `.page-title-action`?

Reply:
I think another link here would still be a button, and we'd want the same styles applied. So I think it would be OK to have it apply to any link contained within `.tec-tickets__modifier-page-title`.